### PR TITLE
Change Cloud Identity to Identity

### PR DESCRIPTION
**Pull request by cyrichardson**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-d813f95d46/docs/cloud-identity/v2/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-e571b84257/docs/cloud-identity/v2/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-ce2225d78b/docs/cloud-identity/v2/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-3829025606/docs/cloud-identity/v2/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-181dd4bf0c/docs/cloud-identity/v2/). :bow:

File: api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst [hunk 1]
```diff
@@ -8,9 +8,9 @@ List extensions
     GET //v2.0/extensions
 
 This operations lists the Identity API extensions that are available in the
-Rackspace Cloud Identity  service. For more information, see the
-:ref:`Extensions <extensions-ovw>` section of the  Rackspace Cloud Identity
-service Developer Guide.
+Rackspace Identity  service. For more information, see the
```
Comment (RackerKate) [inline api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst]: I think there might be an extra space before "service."

File: api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst [hunk 2]
```diff
@@ -8,9 +8,9 @@ List extensions
...
+:ref:`Extensions <extensions-ovw>` section of the  Rackspace Identity
```
Comment (RackerKate) [inline api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst]: I think there might be an extra space before "Rackspace."

File: api-docs/api-reference/token-operations.rst [hunk 3]
```diff
@@ -1,10 +1,10 @@
 .. _token-operations:
 
 =====================
-Authentication Tokens
+Authentication tokens
 =====================
 
-The Rackspace Cloud Identity service service provides
+The Rackspace Identity service service provides
```
Comment (RackerKate) [inline api-docs/api-reference/token-operations.rst]: 2 instances of the word "service."

File: api-docs/getting-started/concepts.rst [hunk 4]
```diff
@@ -137,7 +137,7 @@ Identity service supports two types of roles:
   particular Rackspace Cloud service.
 
 A user with a specific role inherits the rights and privileges associated with
-the role. In Rackspace Cloud Identity service, the authentication token issued
+the role. In Rackspace Identity service, the authentication token issued
```
Comment (RackerKate) [inline api-docs/getting-started/concepts.rst]: Should this be "In the...?"

File: api-docs/getting-started/mfa-content/proc-mfa-enforce.rst [hunk 5]
```diff
@@ -120,7 +120,7 @@ their account.
 Update user-level enforcement policy for multi-factor authentication
 --------------------------------------------------------------------
 
-Rackspace Cloud Identity and user administrators can use customize the
+Rackspace Identity and user administrators can use customize the
```
Comment (RackerKate) [inline api-docs/getting-started/mfa-content/proc-mfa-enforce.rst]: I think there's a typo here at "use customize."

File: api-docs/getting-started/mfa-content/proc-mfa-setup-phone.rst [hunk 6]
```diff
@@ -26,7 +26,7 @@ Cloud Control panel.
     -  When multi-factor is enabled on an account, any existing tokens for
        that user are immediately revoked.
 
-    -  Multi-factor authentication requires Rackspace Cloud Identity
+    -  Multi-factor authentication requires Rackspace Identity
```
Comment (RackerKate) [inline api-docs/getting-started/mfa-content/proc-mfa-setup-phone.rst]: "...requires the...?"

File: api-docs/index.rst [hunk 7]
```diff
@@ -1,28 +1,28 @@
...
+The Rackspace Identity service v2.0 is an implementation of
 OpenStack Keystone Service v2.0 that provides common, token-based
 authentication and authorization services that allow seamless access to
 Rackspace Cloud products and services.
 
 -  Authentication generates a token in response to valid credentials
    submitted by a user, process, or client. Then, that token is used to
    confirm identity when requesting access to systems or resources
-   defined in the Cloud Identity service service catalog.
+   defined in the Rackspace Identity service service catalog.
```
Comment (RackerKate) [inline api-docs/index.rst]: 2 instances of "service."

File: api-docs/index.rst [hunk 8]
```diff
@@ -1,28 +1,28 @@
...
 
 -  Authorization determines the resources and actions available to a
    user, client, or process based on access control capabilities that
-   can be defined and managed using the Rackspace Cloud Identity
+   can be defined and managed using the Rackspace Identity
```
Comment (RackerKate) [inline api-docs/index.rst]: Suggest "...by using..."

File: api-docs/getting-started/manage-auth-tokens.rst [hunk 9]
```diff
@@ -23,10 +23,10 @@ endpoint.
 Best practices
 ~~~~~~~~~~~~~~
 
--  When you authenticate to the Rackspace Cloud Identity service be sure to
+-  When you authenticate to the Rackspace Identity service be sure to
```
Comment (RackerKate) [inline api-docs/getting-started/manage-auth-tokens.rst]: Suggest "...ensure that you cache..."

File: api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst (hunk 1 above)
Comment (cyrichardson) [reply api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst]: Fixed

File: api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst (hunk 2 above)
Comment (cyrichardson) [reply api-docs/api-reference/methods/extensions/get-list-extensions-v2.0.rst]: Fixed

File: api-docs/api-reference/token-operations.rst (hunk 3 above)
Comment (cyrichardson) [reply api-docs/api-reference/token-operations.rst]: Fixed

File: api-docs/getting-started/concepts.rst (hunk 4 above)
Comment (cyrichardson) [reply api-docs/getting-started/concepts.rst]: Fixed

File: api-docs/getting-started/mfa-content/proc-mfa-enforce.rst (hunk 5 above)
Comment (cyrichardson) [reply api-docs/getting-started/mfa-content/proc-mfa-enforce.rst]: Fixed

File: api-docs/getting-started/mfa-content/proc-mfa-setup-phone.rst (hunk 6 above)
Comment (cyrichardson) [reply api-docs/getting-started/mfa-content/proc-mfa-setup-phone.rst]: Fixed

File: api-docs/index.rst (hunk 7 above)
Comment (cyrichardson) [reply api-docs/index.rst]: Fixed

File: api-docs/index.rst (hunk 8 above)
Comment (cyrichardson) [reply api-docs/index.rst]: Fixed

File: api-docs/getting-started/manage-auth-tokens.rst (hunk 9 above)
Comment (cyrichardson) [reply api-docs/getting-started/manage-auth-tokens.rst]: Fixed

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-8f6c001083/docs/cloud-identity/v2/)

File: api-docs/api-reference/methods/multifactor-auth/post-add-an-otp-device-v2.0.rst [hunk 10]
```diff
@@ -7,7 +7,7 @@ Create an OTP device
 
     POST /v2.0/users/{userId}/RAX-AUTH/multi-factor/otp-devices
 
-adds a OTP (one-time password) device for multi-factor authentication
+Adds a OTP (one-time password) device for multi-factor authentication.
```
Comment (RackerKate) [inline api-docs/api-reference/methods/multifactor-auth/post-add-an-otp-device-v2.0.rst]: Maybe "an OTP?"

File: api-docs/general-api-info/svc-catalog-ovw.rst [hunk 11]
```diff
@@ -47,7 +47,7 @@ documentation`_.
 |                        | who request access to Rackspace Cloud service      |
 |                        | access to Rackspace Cloud services.                |
 |                        |                                                    |
-|                        | Cloud Identity is the name of the Rackspace Cloud  |
+|                        | Identity is the name of the Rackspace Cloud        |
 |                        | service of this type. The Rackspace service is     |
```
Comment (RackerKate) [inline api-docs/general-api-info/svc-catalog-ovw.rst]: Should this be "The Rackspace Identity service is..."

File: api-docs/getting-started/concepts.rst [hunk 12]
```diff
@@ -24,7 +24,7 @@ configuration.
 
 In the Rackspace Cloud, users are represented as accounts. The accounts
 are automatically created with a default user type and role, based on the
-Identity Service system configuration. The user type and role determine 
+Identity Service system configuration. The user type and role determine
```
Comment (RackerKate) [inline api-docs/getting-started/concepts.rst]: Should "service" be lowercase here?

File: api-docs/api-reference/methods/multifactor-auth/post-add-an-otp-device-v2.0.rst (hunk 10 above)
Comment (cyrichardson) [reply api-docs/api-reference/methods/multifactor-auth/post-add-an-otp-device-v2.0.rst]: Fixed

File: api-docs/general-api-info/svc-catalog-ovw.rst (hunk 11 above)
Comment (cyrichardson) [reply api-docs/general-api-info/svc-catalog-ovw.rst]: Fixed

File: api-docs/getting-started/concepts.rst (hunk 12 above)
Comment (cyrichardson) [reply api-docs/getting-started/concepts.rst]: Fixed

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-d5d8116162/docs/cloud-identity/v2/)

**Pull request merged**